export.md

```diff
@@ -1,12 +1,13 @@
 # FHIR Bulk Data Export API Proposal
+ 
+## Security Considerations
 
-## Request Flow
+All exchanges described herein between a client and a server MUST be secured using [Transport Layer Security (TLS) Protocol Version 1.2 (RFC5246)](https://tools.ietf.org/html/rfc5246).  Use of mutual TLS is OPTIONAL.  
 
-### Authorization
+All of the requests described herein require client authorization.  Requests sent to a FHIR server (e.g., kick-off request, delete request) and other RESTful services MUST include an OAuth 2.0 access token.  Implementers are encouraged to implement OAuth 2.0 access management in accordance with the [SMART Backend Services: Authorization Guide](https://github.com/smart-on-fhir/fhir-bulk-data-docs/blob/master/authorization.md).  Implementations MAY include non-RESTful services that use authorization schemes other than OAuth 2.0.   
```

Comment:
> . Requests sent to a FHIR server (e.g., kick-off request, delete request) and other RESTful services MUST include an OAuth 2.0 access token.

This implies the use of OAuth with the Bulk Export API, but I don't see why we should constrain in this way. Obviously if you're using SMART Backend Services, *that's* an OAuth spec; but otherwise, you might use this with, say, mutual TLS and no further client authentication. Why constrain it/